config/environments/production.rb

```
@@ -67,6 +67,18 @@
   # want to log everything, set the level to "debug".
   config.log_level = ENV.fetch("RAILS_LOG_LEVEL", "info")
 
+  if ENV['SENDGRID_API_KEY'].present?
+    ActionMailer::Base.smtp_settings = {
+      user_name: 'apikey', # This is the string literal 'apikey', NOT the ID of your API key
+      password: ENV['SENDGRID_API_KEY'], # This is the secret sendgrid API key which was issued during API key creation
+      domain: Rails.application.routes.default_url_options[:host]
+      address: 'smtp.sendgrid.net',
```

RuboCop (1.73)

[fatal] unexpected token tIDENTIFIER
(Using Ruby 3.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)